src/plays/currencyconverter/styles.css

```diff
@@ -0,0 +1,3 @@
+.bg-img {
```

Comment:
Please rename the class to a unique name so the website does not break.

Reply:
resolve ui issue with others